telegram/_poll.py

```diff
@@ -146,7 +146,7 @@ class Poll(TelegramObject):
         id (:obj:`str`): Unique poll identifier.
         question (:obj:`str`): Poll question, :tg-const:`telegram.Poll.MIN_QUESTION_LENGTH`-
             :tg-const:`telegram.Poll.MAX_QUESTION_LENGTH` characters.
-        options (Sequence[:class:`PollOption`]): List of poll options.
+        options (Sequence[:class:`~telegram.PollOption`]): List of poll options.
```

Comment:
just double checking: These changes are not strictly needed anymore, right? I'm okay with keeping them to avoid having to revert though :)

Reply:
That is correct